input/fsh/QuestionnaireResponseSEVendorLite.fsh

```diff
@@ -0,0 +1,59 @@
+Profile: QuestionnaireResponseSEVendorLite
+Parent: QuestionnaireResponse
+Description: "Questionnaire response with restrictions for swedish vendors."
+* identifier 1.. MS
+* subject only Reference(PatientSEVendorLite)
+* questionnaire only Canonical(QuestionnaireSEVendorLite)
+* item.answer.value[x] only string
```

Comment:
only string response are expecting? can be boolean as well for some yes/no questions. :-)